action.yml

```diff
@@ -114,6 +114,7 @@ runs:
     - name: Validate markup
       run: |
         echo "::group::Validate markup"
+        echo "::add-matcher::.github/workflows/vnu-jar-problem-matcher.json"
```

Comment:
```suggestion
        echo "::add-matcher::${{ github.action_path }}/.github/workflows/vnu-jar-problem-matcher.json"
```

Reply:
Or maybe we should register it in the JS file if `@actions/core` supports it.

Reply:
Should probably store the matcher at `src/validate-markup-problem-matcher.json` though.
(I've plans to restructure the files into directories in `/src`)

Reply:
OK, I'll move it over. It seems like it wants a file/path rather than just passing a JSON object https://github.com/actions/toolkit/blob/dd046652c32b45c0207b37ee0bc0ebf43a28b257/docs/problem-matchers.md

Reply:
Actually ended up taking the approach from setup-go and registering the file through the API, but it still seems to need the actual physical file

Reply:
No worries on additional file :)